.gitlab/packaging/oci.yml

```
@@ -1,25 +1,18 @@
-packaging_oci:
+.package_oci:
   stage: packaging
   rules:
     - !reference [.except_mergequeue]
     - when: on_success
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64$DATADOG_AGENT_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_BUILDIMAGES
   tags: ["arch:amd64"]
-  needs:
-    [
-      "agent_remote_updater-x64-a7",
-      "agent_remote_updater-arm64-a7",
-    ]
-  variables:
-    KUBERNETES_CPU_REQUEST: 16
-    KUBERNETES_MEMORY_REQUEST: "32Gi"
-    KUBERNETES_MEMORY_LIMIT: "32Gi"
+  before_script:
+    - source /root/.bashrc
+    - export INSTALL_DIR=/opt/datadog-packages/datadog-agent/$(inv agent.version -u)-1
```

```suggestion
    - export INSTALL_DIR=/opt/datadog-packages/datadog-agent/$(inv agent.version -u --no-include-git --no-include-pre)-1
```
Not sure about this though..
Bonus: you can maybe also `export PACKAGE_VERSION` on `before_script` and use it in the definition of `INSTALL_DIR` if it's the one you really need, wdyt?

---

Plus, should it be `export INSTALL_DIR=/opt/datadog-packages/${OCI_PRODUCT}/$(inv agent.version -u)-1` instead?

---

Having `--no-include-git` and `--no-include-pre` is likely to cause conflicts for dev builds as they will all share the same version no?
You are 100% correct regarding the `${OCI_PRODUCT}` part, I'll fix this right away, and will take a look at your suggestion for `PACKAGE_VERSION`

---

Done!